lib/outbrain/api/marketer.rb

```diff
@@ -19,6 +20,12 @@ def self.create(*)
       def budgets
         Budget.find_by(marketer_id: id)
       end
+
+      def campaign_reports(from, to)
+        CampaignReport.where(id, from, to)
+      end
+
+      class CampaignReport < Report; end
```

Comment:
like it, very sneaky but have a feeling this will deserve it's own file when you start moving the path :-)